src/core/optimization_container.jl

```diff
@@ -48,6 +48,27 @@ function Base.isempty(pvc::PrimalValuesCache)
     return isempty(pvc.variables_cache) && isempty(pvc.expressions_cache)
 end
 
+mutable struct ObjectiveFunction
```

Comment:
I don't see this being used or tested. Expected?

Reply:
It is covered by all the uses of the optimization container.